LoopKitUI/Views/ExpandableDatePicker.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  ExpandableDatePicker.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 8/12/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ExpandableDatePicker: View {
```

Comment:
Created 'custom' date picker because the current SwiftUI one doesn't expand properly if it's not in a Form (which we can't do here)